app/src/androidTest/java/ch/epfl/sdp/drone3d/drone/DroneServiceTest.kt

```diff
@@ -45,6 +52,8 @@ class DroneServiceTest {
 
         verify(factory).createSimulation(ip, port)
 
+        // now stop isConnected to have to return true
+        `when`(drone.core.connectionState).thenCallRealMethod()
```

Comment:
I am not sure if this is needed as disconnect will set the drone instance to null and DroneData will set isConnected to false